src/integration/billing/stripe/StripeBillingIntegration.ts

```diff
@@ -1005,7 +1005,7 @@ export default class StripeBillingIntegration extends BillingIntegration {
       customer: user.billingData.customerID,
       status: BillingInvoiceStatus.OPEN,
     });
-    if (list && list.data && list.data.length > 0) {
+    if (list && !Utils.isEmptyArray(list.data)) {
```

Comment:
You can even do:

`if (list && !Utils.isEmptyArray(list.data)) {` -> `if (!Utils.isEmptyArray(list?.data)) {`